js/script.js

```diff
@@ -0,0 +1,59 @@
+const hambIcon = document.getElementById('hamburgerIcon');
+const closeIcon = document.getElementById('closeIcon');
```

Comment:
- [ ] I would suggest you make your `X` buttons to match the figma design
### your design
![your](https://user-images.githubusercontent.com/81727988/179810162-2981eca6-47dd-4b08-8f1b-2d674cf94b1e.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/179810229-038e5c76-d786-468b-a505-cb27eb6fb758.png)

Reply:
Updated to font-awesome's latest version (6.1.1), and adapted the icon's styling